CHANGELOG.rst

```diff
@@ -8,31 +8,35 @@ Notable project changes since release 1.3.1 (2017-02-22).
 Unreleased
 ==========
 
+
+20.1 (2020-07-22)
+=================
+
 Added
 -----
 
-- REKS calculations for a strongly correlated system
+- REKS (REstricted Kohn-Sham) calculations for ground and low-lying exited
+  states
```

Comment:
The accurate abbreviation for REKS is spin-Restricted Ensemble Kohn-Sham, so please fix this.

Reply:
@inseonglee apologies, now corrected.